flink-runtime/src/test/java/org/apache/flink/runtime/util/RunnablesTest.java

```diff
@@ -53,9 +55,9 @@ public void testExecutorService_uncaughtExceptionHandler() throws InterruptedExc
                 () -> {
                     throw new RuntimeException("foo");
                 });
-        Assert.assertTrue(
-                "Expected handler to be called.",
-                handlerCalled.await(TIMEOUT_MS, TimeUnit.MILLISECONDS));
+
+        // expect handler to be called
+        handlerCalled.await();
```

Comment:
Why did you not apply this to the `testScheduledExecutorService_uncaughtExceptionHandler` as well?

Reply:
Thanks. I wasn't aware of this behavior of the `ScheduledExecutorService` and I didn't read the `Expected handler not to be called.` message of the assertion properly.